bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigDescriptionParameter.java

```diff
@@ -81,6 +83,8 @@
     private String unitLabel;
 
     private String context;
+
+    @SerializedName("default")
```

Comment:
Do you have any explanation why this is required? Are we some where serializing ConfigDescriptionParameter instead of ConfigDescriptionParameterDTO?